enterprise/api/v1/serializers.py

```diff
@@ -1735,7 +1735,7 @@ class EnterpriseGroupLearnersRequestQuerySerializer(serializers.Serializer):
     show_removed = serializers.BooleanField(required=False, default=False)
     is_reversed = serializers.BooleanField(required=False, default=False)
     page = serializers.IntegerField(required=False)
-    lms_users = serializers.ListField(
-        child=serializers.IntegerField(required=True),
+    learners = serializers.ListField(
+        child=serializers.EmailField(required=True),
```

Comment:
[nit] I think the `required` field could be set to `False`?

Reply:
The email fields are required for the `learners` query param, the `learners` query param is not required for a request, which should be reflected by our tests